move to use registry.k8s.io (#2169)
kubernetes/cloud-provider-openstack: docs/cinder-csi-plugin/using-cinder-csi-plugin.md

## Downloads

> Stable released version images of the plugin can be found at [Github](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-provider-os/images.yaml)

Reviewer: ```suggestion
Stable released version images of the plugin can be pulled from `registry.k8s.io/provider-os/cinder-csi-plugin:[release tag]`
```
